.github/workflows/reuse.yml

```diff
@@ -0,0 +1,254 @@
+name: If Levaithan was reusuable?
+
+on:
+  workflow_call:
+    inputs:
+      LEVIATHAN_REF:  # id of input
+        description: "Leviathan branch, or tag, or commit to test with. Examples: 'refs/heads/<branchName>', 'refs/tags/<tagName>', '<commitId>'"
+        required: false
+        type: string
+        default: 'refs/heads/master'
+      # CONFIG:
+      #   description: 'path to config.js'
+      #   required: false
+      #   default: './suites/config.js'
+      SUITES:
+        description: 'path to test folders'
+        required: false
+        type: string
+        default: './suites'
+      # TEST_SUITE_NAME:
+      #   description: 'which test suite to run'
+      #   required: false
+      #   type: string
+      WORKSPACE:
+        description: 'path to WORKSPACE'
+        required: false
+        type: string
+        default: './.leviathan-inputs'
+      DEVICE_TYPE:
+        description: "slug of device type"
+        required: true
+        type: string
+      REPORTS:
+        description: "path to reports directory"
+        required: false
+        type: string
+        default: "./reports"
+      ENVIRONMENT:
+        description: "What BalenaCloud env, balena-cloud or balena-machine"
+        required: true
+        type: string
+        default: "balena-cloud"
+      BALENACLOUD_ORG:
+        required: false
+        description: "Organization used during cloud suite to create fleets, etc."
+        default: "testbot"
+        type: string
+      BALENACLOUD_APP_NAME:
+        description: "Testbot fleet for finding available Leviathan workers. Not used for QEMU workers. Can accept a list of apps separated by commas."
+        type: string
+        required: false
+        default: "balena/testbot-rig"
+      BALENACLOUD_API_URL:
+        type: string
+        required: false
+        default: "balena-cloud.com"
+      BALENAMACHINE_API_URL:
+        required: false
+        type: string
+        default: "bm.balena-dev.com"
```

Comment:
Same as above, we shouldn't need separate inputs for ENVIRONMENT, BALENACLOUD_API_URL,   BALENAMACHINE_API_URL.

Reply:
We do not. But, it's mode switch basically. Will discuss.

Reply:
Should we not use github environments for this, like we are with staging/prod on the yocto action? 

Because what we have is differing API_URL , SSH_PORT , SSH_URL, depending on the environment target

Reply:
Can be done, I did look at them but didn't full explore their usage and yes, that's exactly what we are using these for.